src/main/java/com/google/sps/servlets/AuthorsServlet.java

```diff
@@ -0,0 +1,91 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Resources;
+import com.google.sps.data.Author;
+import com.hubspot.jinjava.Jinjava;
+import com.hubspot.jinjava.JinjavaConfig;
+import com.hubspot.jinjava.loader.FileLocator;
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet(urlPatterns = {"/authors"})
+public class AuthorsServlet extends HttpServlet {
+
+  private static final Collection<Author> authors = new ArrayList<>(3);
+
+  @Override
+  public void init() {
+    authors.add(
+        new Author(
+            "Thomas Quintanilla",
+            "Duke University",
+            "I'm a kool boi. I do computer science things. I make cool mp3 and me project. "
+                + "I love mentorship, and I am a passionate boi.",
+            "thomas.jpg"));
+    authors.add(
+        new Author(
+            "Sylvia Ziyu Zhang",
+            "Carnegie Mellon University",
+            "I'm a kool gal. I do computer science things. I do cool natural language processing things. "
+                + "I am a teaching assistant, and I am a passionate gal.",
+            "sylvia.jpg"));
+    authors.add(
+        new Author(
+            "Mudit Gupta",
+            "Georgia Institute of Technology",
+            "I'm a kool boi. I do computer science things. I make cool visualization project. "
+                + "I love mentoring, and I am a passionate boi.",
+            "mudit.jpg"));
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    System.out.println("REQUEST AT: " + request.getServletPath());
+
+    response.setContentType("text/html;");
+
+    JinjavaConfig config = new JinjavaConfig();
+    Jinjava jinjava = new Jinjava(config);
+    try {
+      jinjava.setResourceLocator(
+          new FileLocator(new File(this.getClass().getResource("/templates").toURI())));
+    } catch (URISyntaxException e) {
+      System.err.println("templates dir not found!");
+    }
+
+    Map<String, Object> context = new HashMap<>();
+    context.put("url", "/authors");
+    context.put("authors", authors);
+
+    String template =
+        Resources.toString(this.getClass().getResource("/templates/authors.html"), Charsets.UTF_8);
```

Comment:
will be better if we move all string constants that reference resources to a single resource constants file

Reply:
once the "create resource string constants class" PR is merged, I'll rebase and use that here